docs/source/spec/core.rst

```diff
@@ -505,6 +504,13 @@ definition using an ``apply`` statement:
             }
         }
 
+The member of a list cannot directly or transitively target the list
```

Comment:
Should similar paragraphs be added for set and map types?

Reply:
Also a removed leftover from a previous iteration